api/util.ts

```diff
@@ -77,3 +77,16 @@ export function stringify(data: any) {
 
   return JSON.stringify(data) + ''
 }
+
+/**
+ * Returns a Promise that will resolve after the number of seconds provided. Useful
+ * for keeping the async/await patterns in functions that require a simple sleep.
+ * 
+ * @param numberOfSeconds the number of seconds that this function will take to exit.
+ * @return {Promise<void>} a Promise that will resolve after the number of seconds provided.
+ */
+export function timeout(numberOfSeconds:number) : Promise<void>{
```

Comment:
This is duplicate of the `delay` method above, which takes milliseconds instead of seconds